go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/attwad/nessie
+
+go 1.15
```

Comment:
Can you change the github actions workflow to build using go 1.15 as well please?
https://github.com/attwad/nessie/blob/master/.github/workflows/go.yml#L10